src/app/components/ItemPage/ItemHoldings.jsx

```diff
@@ -36,55 +36,45 @@ class ItemHoldings extends React.Component {
     const collapsed = !this.state.expanded;
 
     return (
-      <ul>
-        {
-          holdings.map((h, i) => {
-            let itemLink;
-            let itemDisplay = null;
-
-            // if (!h.available) {
-            //   itemLink = <span className="nypl-item-unavailable">{h.accessMessage}</span>;
-            // } else if (h.isElectronicResource) {
-            //   itemLink = <a href={h.url}>View Online</a>;
-            // } else {
-              // NOTE: This is using `this.props.bibId` but it is wrong. It should be the item ID.
-              // Currently, hitting the API with items is not working.
+      <table className="nypl-basic-table">
```

Comment:
`hidden` works these days, so we can swap out `visuallyHidden` where we see it when refactoring. Updating now...